### ISSUE TITLE
Can't create gam project. ERROR: 403: Permission iam.serviceAccountKeys.create is required to perform this operation on service account

### ISSUE THREAD

**Issue by lucascantor** [jay0lee/GAM#1382]
Confirming the following:
* I have upgraded to the latest GAM release from https://git.io/gamreleases and I still have this issue - YES
* I am typing the command as described in the GAM Wiki at https://github.com/jay0lee/gam/wiki - YES

Full steps to reproduce the issue:
1. Ensure GAM is installed and up-to-date
1. Attempt to set up a Google API project for GAM

Expected outcome (what are you trying to do?):
- Successful GAM project creation

Actual outcome (what errors or bad behavior do you see instead?):
```
GAM is now installed. Are you ready to set up a Google API project for GAM? (yes or no) yes
Please enter your Google Workspace admin email address: [REDACTED]

Go to the following link in your browser:

	https://gam-shortn.appspot.com/[REDACTED]

Enter verification code: [REDACTED]
Creating project "GAM Project"...
Checking project status...
  Project: gam-project-[REDACTED], Enable 21 APIs
    API: admin.googleapis.com, Enabled (1/21)
    API: alertcenter.googleapis.com, Enabled (2/21)
    API: calendar-json.googleapis.com, Enabled (3/21)
    API: chat.googleapis.com, Enabled (4/21)
    API: chromemanagement.googleapis.com, Enabled (5/21)
    API: chromepolicy.googleapis.com, Enabled (6/21)
    API: classroom.googleapis.com, Enabled (7/21)
    API: cloudidentity.googleapis.com, Enabled (8/21)
    API: contacts.googleapis.com, Enabled (9/21)
    API: drive.googleapis.com, Enabled (10/21)
    API: driveactivity.googleapis.com, Enabled (11/21)
    API: iap.googleapis.com, Enabled (12/21)
    API: gmail.googleapis.com, Enabled (13/21)
    API: groupssettings.googleapis.com, Enabled (14/21)
    API: iam.googleapis.com, Enabled (15/21)
    API: licensing.googleapis.com, Enabled (16/21)
    API: reseller.googleapis.com, Enabled (17/21)
    API: sheets.googleapis.com, Enabled (18/21)
    API: siteverification.googleapis.com, Enabled (19/21)
    API: storage-api.googleapis.com, Enabled (20/21)
    API: vault.googleapis.com, Enabled (21/21)
Setting GAM project consent screen...
Creating Service Account
 Generating new private key...
 Extracting public certificate...
 Done generating private key and public certificate.
 Uploading new public certificate to Google...

ERROR: 403: Permission iam.serviceAccountKeys.create is required to perform this operation on service account projects/-/serviceAccounts/111146252721742176654. - 403
Project creation failed. Trying again. Say N to skip project creation.
```

**Comment by lucascantor** [comment]
This appears to be impacting others, as seen at https://groups.google.com/g/google-apps-manager/c/kKfd7lVIVaQ/m/3XvD5am2CwAJ

**Comment by jay0lee** [comment]
Seems to be a timing issue, the key upload succeeds if you try again a few seconds later. No idea why Google servers suddenly need more time before they honor the permissions but I'll get a patched build out soon to fix this.

**Comment by jay0lee** [comment]
This is now fixed in GAM 6.03.
